dramatiq/cli.py

```diff
@@ -118,7 +118,7 @@ def folder_path(value):
     return os.path.abspath(value)
 
 
-def parse_arguments():
+def argument_parser():
```

Comment:
Is there a particular reason to change this function? Do you plan to augment the argument parser?

Reply:
Wlel, the function first parsed the arguments and returned them. Now it returns the argument parser itself. 

It think it should be clear what the function is doing, that's why I suggest to change the name.

Reply:
Right, I wasn't asking about the name specifically. I was asking about the reasoning behind changing it to return the parser. :D

Reply:
Oh, sorry for the misunderstanding 😃 I have two existing scenarios in our project where we would like to pass the arguments to the main function instead of parsing the arguments inside it.

1) A custom entrypoint (`console_script`) with predefined arguments, see the example in the description of this pull request
2) A custom entrypoint (`console_script`) which does not parse the arguments from the commandline but from a configuration file deployed with our configuration manager

Now I have to mokeypatch the `main` function in `cli.py` to do so because there the parsing happens.